### PR TITLE
dart 2.1

### PR DESCRIPTION
**Pull request by Cheackraze**
## About the PR 
The behaviour of the Dart shuttle was not ideal since I just copied over the old dart shuttle prototype and components. This removes the BecomesStation component so that it no longer shows up in the late join list if spawned in.  Also removes the availableJobs and replaces it with 1 captain, similar to the other shuttles infiltrator and pirate ship.

Ive had reports that light fixtures arent filling when admins are spawning the shuttle in but I cant reproduce it, its working fine locally with loadmap, loadgamemap, and mapping/mapinit commands

**Changelog**

stops the Dart ERT shuttle from showing up in the latejoin menu

### CODE REVIEW

**Comment by Cheackraze** [comment]
looks like the tests come back as not happy when it doesn't have the BecomesStation component, but that's the one that causes it to show up in the latejoin menu and it confuses people.

File: Resources/Prototypes/Maps/dart.yml [hunk 1]
```diff
@@ -4,15 +4,11 @@
...
         !type:NanotrasenNameGenerator
         prefixCreator: '14'
       overflowJobs: []
       availableJobs:
-        ERTLeader: [ 0, 1 ]
-        ERTEngineer: [ 0, 1 ]
-        ERTSecurity: [ 0, 1 ]
-        ERTMedical: [ 0, 1 ]
-        ERTJanitor: [ 0, 1 ]
+        Captain: [ 1, 1 ]
```
Comment (moonheart08) [inline Resources/Prototypes/Maps/dart.yml:14]: Remove this outright and have the stations line just be
`stations: {}`

**Comment by Cheackraze** [comment]
alright the yml was tidied up but the tests are still angry because I took off the BecomesStation component because it wants dart to have a station grid

**Comment by AJCM-git** [comment]
tests are still failing, have you figured out why?

**Comment by Cheackraze** [comment]
> tests are still failing, have you figured out why?

its because it doesn't have a station component. I will try to mess with it tomorrow, see if I can save it as a grid while keeping it available for admins to spawn via loadgamemap.  The issue is that it originally was a station with jobs for auto spawning, after it was reworked it requires the station component still.  The problem is, it shows up in the late join menu as an empty entry which confuses people.

**Comment by Cheackraze** [comment]
> tests are still failing, have you figured out why?

its because the tests are trying to run LoadGameMap tests, which is trying to ensure a playable station. This is just a shuttle and needs to be tested with LoadGrid, like the evac shuttles and stuff are but I don't know what to do to fix

**Comment by AJCM-git** [comment]
> This is just a shuttle and needs to be tested with LoadGrid, like the evac shuttles and stuff are but I don't know what to do to fix

Thats the problem, the evac shuttles arent gameMaps, the test fails because as you said its trying to ensure the map is valid, so you should just ditch the map prototype in here and the wizard shuttle since `loadgrid` doesnt take a map id as argument anyways, it takes the path to the grid prototype.
Not to say this shouldnt be supported but as things are now you have to either make them proper game maps or grids as expected by tests

**Comment by Cheackraze** [comment]
Ditching the prototype means it cannot be spawned by admins.... defeating the entire purpose of having the shuttle to begin with

**Comment by AJCM-git** [comment]
`loadmap` command takes a path

**Comment by Cheackraze** [comment]
> `loadgrid` command takes a path

The admins don't use load grid, they don't have access to load arbitrary grids, they use loadgamemap

This is like the 4th time I've had to explain this!

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**